### PR TITLE
Add new page for interactors+storybook introduction

### PR DESCRIPTION
**Pull request by minkimcello**
## Motivation

To update interactors docs to include storybook + interactors.

## Approach

- Created a new separate page instead of putting it in with `Jest & Cypress`. Although the three all fall under "integrations", storybook isn't a testing tool.
- See preview [here](https://deploy-preview-967--bigtest.netlify.app/bigtest/docs/interactors/storybook). I put in a very simple example but I'm not sure if it's correct.
- I also may have missed some of the major selling points of using interactors + storybook.

### CODE REVIEW

File: website/docs/interactors/8-storybook.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import userEvent from '@testing-library/user-event';
+
+export const FormSignIn = {
+  play: () => {
+    userEvent.type(screen.getById('email'), 'homer@gmail.com');
+    userEvent.type(screen.getById('password'), 'donuts123');
+    userEvent.click(screen.getByText('Sign In'));
+  }
+};
+```
```
Comment (wKich) [inline website/docs/interactors/8-storybook.md]: I suggest swapping examples. It's better to say, how Storybook suggests writing `play` methods and how they might look better with interactions.

File: website/docs/interactors/8-storybook.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+```js
+import { screen } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+export const FormSignIn = {
+  play: () => {
+    userEvent.type(screen.getById('email'), 'homer@gmail.com');
+    userEvent.type(screen.getById('password'), 'donuts123');
+    userEvent.click(screen.getByText('Sign In'));
+  }
```
Comment (wKich) [inline website/docs/interactors/8-storybook.md]:
```suggestion
  play: async () => {
    await userEvent.type(screen.getById('email'), 'homer@gmail.com');
    await userEvent.type(screen.getById('password'), 'donuts123');
    await userEvent.click(screen.getByText('Sign In'));
  }
```

**Review by wKich — COMMENTED** [review]
@cowboyd Do we want to say about future features and possibilities, like testing here?

**Comment by cowboyd** [comment]
> @cowboyd Do we want to say about future features and possibilities, like testing here?

It's possible, but it's kinda fuzzy in my mind what the possibilities are and so I'd say let's propose something to add here with as the story develops. Overall though, I'm inclined to roll with what we have and add to it later.

**Review by cowboyd — APPROVED** [review]
Is there any reason not to add this right now?

**Pull request merged**